src/Viewer.tsx

```diff
@@ -663,6 +666,8 @@ function Viewer(): ReactElement {
     },
   ];
   const tabItems = allTabItems.filter((item) => item.visible !== false);
+  const visibleTabKeys = useMemo(() => new Set(tabItems.map((item) => item.key)), [INTERNAL_BUILD]);
+  const currentTab = visibleTabKeys.has(openTab) ? openTab : TabType.TRACK_PLOT;
```

Comment:
Fixes a bug where a link could specify a tab that isn't currently visible (due to internal vs. public build modes)